modules/101-cert-manager/openapi/config-values.yaml

```diff
@@ -101,3 +101,10 @@ properties:
     description: |
       The email used for sending notifications by LetsEncrypt.
     x-examples: ["example@example.com"]
+
+  certificateOwnerRef:
```

Comment:
I would like to see a more concrete name connected to what this option is doing, e.g., `cleanupOrphanSecrets`.

Reply:
Thinking about naming,

> Field and resource names should be declarative, not imperative (SomethingDoer, DoneBy, DoneAt).

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions

It's probably not that easy to use declarative name, like `secretDeletion: true`, or `referencedSecretCleanup: true`.